swarm/api/http/server.go

```diff
@@ -644,6 +653,8 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
+
+
```

Comment:
too many empty lines here maybe?

Reply:
Yes, please delete these extra lines that were added.